.vscode/settings.json

```diff
@@ -22,6 +22,7 @@
     "packages/node_modules/**": false
   },
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": true,
+    "source.organizeImports": false
```

Comment:
I was thinking, Should we commit ```.vscode/settings.json``` this file..

Reply:
This wont affect us , its just makes sure that if we had bad vscode config it simply wont re arrange the imports 

we will need to upgrade the eslint in our project and add the import rules there 

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-321260